R/class2tree.R

```diff
@@ -201,10 +206,11 @@ taxa2dist <- function(x, varstep = FALSE, check = TRUE, labels) {
 mainTaxonomyRank <- function() {
   return(
     c(
-      "strain","biotype","isolate","pathogroup","serogroup","serotype",
-      "genotype","morph","forma","subspecies","subvariety","varietas",
-      "formaspecialis","subspecies","species","speciessubgroup",
-      "speciesgroup","series","subgenus","genus","subtribe","tribe",
+      "isolate","strain","pathogroup","serotype","serogroup",
```

Comment:
@trvinh can you put this change on a different git branch? i assume this is the commit in this issue https://github.com/ropensci/taxize/issues/875 - since they are unrelated I'd prefer they were kept separate

Reply:
@skott: I have added a test and reverted the commit related to #875